torch/nn/modules/activation.py

```diff
@@ -208,7 +208,7 @@ def __repr__(self):
 
 class Tanh(Module):
     r"""Applies element-wise,
-    :math:`f(x) = (exp(x) - exp(-x)) / (exp(x) + exp(-x))`
+    :math:`f(x) = \frac{\exp(x) - \exp(-x)} {\exp(x) + \exp(-x)}`
```

Comment:
Can you please change `f(x)` everywhere to match the name of the module (`\tanh` in this case)?

Reply:
To be honest, I'd argue that the `f(x)` on the left hand side is clearer, but we should simply write `tanh(x)`, on the right hand side, instead of the huge Clover-compatible equivalence?